go/private/actions/link.bzl

```diff
@@ -83,7 +83,9 @@ def emit_link(
         "{}={}={}".format(d.label, d.importmap, d.file.path)
         for d in test_archives
     ])
-    builder_args.add(dep_args, before_each = "-dep")
+    depsFile = go.actions.declare_file(executable.basename + ".deps", sibling = executable)
```

Comment:
I think there are two better potential solutions for this:

1. Ideal long term solution: we should write an importcfg file. We no longer support Go 1.8, so nothing is blocking this. The main issue is that for error reporting, we should include the Bazel label for each package. I guess we could write that in a comment within the importcfg file.
2. Short term solution: we could use [Bazel's support for param files](https://docs.bazel.build/versions/master/skylark/lib/Args.html#use_param_file). Some common functionality could be added in go/tools/builders/env.go so that all of the builders would benefit from this.

Reply:
Done. Updated the change to use bazel's params files.